tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
I thought I have included a tsconfig file in my PR, I might have forgot to add that. Can we set strict to true and see if anything fails?

Reply:
yes the file was not committed. I can set to true, build seems to be ok